src/Benchmarks/Data/RawDb.cs

```diff
@@ -39,6 +39,9 @@ public RawDb(IRandom random, DbProviderFactory dbProviderFactory, IOptions<AppSe
         
         async Task<World> ReadSingleRow(DbConnection connection, DbCommand cmd)
         {
+            // Prepared statements improve PostgreSQL performance by 10-15%
+            cmd.Prepare();
```

Comment:
Does this work?  I needed to put it after db.OpenAsync when I was testing with fortunes.